doc/parameters.md

```diff
@@ -240,9 +240,10 @@ This is the value that will be used in the nginx `worker_processes` [directive](
 
 ### `BACKEND_ENDPOINT_OVERRIDE`
 
-URI that overrides backend endpoint from the configuration. Useful when deploying outside OpenShift deployed AMP.
+URI that overrides the backend endpoint. By default, it is the external route. 
```

Comment:
I'm not sure it's a good idea to focus so much on Openshift given that this is valid for any kind of deployment.

Reply:
That's the whole point of this PR as I see it. Could you please share what other use cases there are for this parameter then? Also if it's not on OpenShift I don't see when else this would be used.

Reply:
Local development for example. Or exposing 3scale backend via a different route. Using geo routing, ...